plugins/modules/ios_service.py

```diff
@@ -0,0 +1,425 @@
+#!/usr/bin/python
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+The module file for ios_service
+"""
+from __future__ import absolute_import, division, print_function
+
+
+__metaclass__ = type
+DOCUMENTATION = """
+---
+module: ios_service
+short_description: Resource module to configure service.
+description:
+  - This module configures and manages service attributes on IOS platforms
+version_added: 1.0.0
```

Comment:
This would be the latest cisco.ios version

Reply:
I have put the current version (`4.5.0`) but should I put `4.6.0` or a tag like `latest` ?